doc/src/vpr/file_formats.rst

```diff
@@ -738,6 +738,73 @@ An example listing for a global net is given below.
+   hh_core:hh_core_0|cycles_lo[0]  53   24     0       19     # 86 dffeas
+   hh_core:hh_core_0|cycles_lo[1]  53   24     0       18     # 86 dffeas
+   hh_core:hh_core_0|cycles_lo[2]  53   24     0       17     # 86 dffeas
+   hh_core:hh_core_0|cycles_lo[3]  53   24     0       16     # 86 dffeas
+   hh_core:hh_core_0|cycles_lo[4]  53   24     0       15     # 86 dffeas
+   hh_core:hh_core_0|cycles_lo[5]  53   24     0       14     # 86 dffeas
+   hh_core:hh_core_0|cycles_lo[6]  53   24     0       13     # 86 dffeas
+   # below is the root of a carry chain segment
+   hh_core:hh_core_0|Add0~2        53   24     0        0     # 86 lcell_comb
+   # below is the rest of the carry chain segment (ignored by the VPR legalizer)
+   hh_core:hh_core_0|Add0~6        53   24     0        1     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~10       53   24     0        2     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~14       53   24     0        3     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~18       53   24     0        4     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~22       53   24     0        5     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~26       53   24     0        6     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~30       53   24     0        7     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~34       53   24     0        8     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~38       53   24     0        9     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~42       53   24     0       10     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~46       53   24     0       11     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~50       53   24     0       12     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~54       53   24     0       13     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~58       53   24     0       14     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~62       53   24     0       15     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~66       53   24     0       16     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~70       53   24     0       17     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~74       53   24     0       18     # 86 lcell_comb
+   hh_core:hh_core_0|Add0~78       53   24     0       19     # 86 lcell_comb
+
+For complex pack molecules such as carry chains (see :ref:`Pack Patterns in <arch_complex_blocks>`), it is only necessary to include the root primitive in the flat placement file; the legalizer ignores non-root primitives and places all primitives in each pack molecule based on the root primitive entry. In the example file above, all lcell_comb instances shown are part of a carry chain segment, and so all except the root primitive are ignored.
+VPR's legalizer does not require the flat placement file to contain every primitive in the netlist; it will reconstruct the portion of the netlist and print a warning if the file does not cover the entire netlist. This functionality is useful for debugging legalization in an external placement tool.
```

Comment:
"reconstruct the portion of the netlist" -> "reconstruct the packing and placement for the specified portion of the netlist"